docs/status.rst

```diff
@@ -48,12 +57,11 @@ A summary of the on-going status of the ``hls4ml`` tool is in the table below.
 
 Other feature notes:
 
-* ``hls4ml`` is tested on Linux, and supports Vivado HLS versions 2018.2 to 2020.1. Vitis HLS is not yet supported. Windows and macOS are not supported.
+* ``hls4ml`` is tested on Linux, and supports Vivado HLS versions 2018.2 to 2020.1 and Intel HLS versions XXX. Vitis HLS is experimentally supported in v0.7.0. Windows and macOS are not supported.
```

Comment:
We use 20.1 to 21.4 of Intel HLS at fnal. Should I put those numbers? Also, macs are partially supported--not for synthesis of course. But maybe we don't need to claim it?

Reply:
yes, please add the numbers. for now i wouldn't mention mac